content/en/blog/2019/v1beta1-authorization-policy/index.md

```diff
@@ -0,0 +1,328 @@
+which will completely replace the previous `v1alpha1` RBAC policy going forward. The `v1beta1` policy provides
+more features but is not backward compatible and will require manual conversion from the old API (we
+will provide tool to help automate the conversion).
+
+In this article, we will introduce the new `v1beta1` authorization policy, talk about its design principles
+and the migration from `v1alpha1` RBAC policy.
+
+## Design principles
+
+A few key design principles played a role in the `v1beta1` authorization policy:
+
+* Align with [Istio Configuration Model](https://goo.gl/x3STjD) for better clarity on the policy target.
+The configuration model provides a unified way for configuration hierarchy, resolution, target selection and etc.
+
+* Improve user experience by simplifying the API. It's much easier to manage one custom resource
+definition (CRD) that includes all access control specification instead of looking at multiple
+different CRDs.
+
+* Support more use cases without introducing much complexity. For example, allow the policy
+to be applied on Ingress/Egress gateway to enforce access control for traffic entering/exiting the mesh.
+
+## `AuthorizationPolicy`
+
+An [`AuthorizationPolicy` custom resource](/docs/reference/config/authorization/authorization-policy/)
+enables access control on workloads. This section gives a high level overview of the
+new changes in the `v1beta1` authorization policy.
+
+An `AuthorizationPolicy` includes a `selector` and a list of `rule`. The `selector`
+is used to decide where to apply the policy, the `rule` is used to decide **who**
+could do **what** under which **conditions**.
+
+The `selector` replaces the functionality provided by `ClusterRbacConfig`
+and the `services` field in `ServiceRole`. The `rule` includes `from`, `to` and
+`when` which replaces the other fields in `ServiceRole` and `ServiceRoleBinding`.
+
+### Example
+
+The following authorization policy applies to workloads with label `app: httpbin`
+and `version: v1` in namespace `foo`:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+ name: httpbin
+ namespace: foo
+spec:
+ selector:
+   matchLabels:
+     app: httpbin
+     version: v1
+ rules:
+ - from:
+   - source:
+       principals: ["cluster.local/ns/default/sa/sleep"]
+   to:
+   - operation:
+       methods: ["GET"]
+   when:
+   - key: request.headers[version]
+     values: ["v1", "v2"]
+{{< /text >}}
+
+The policy allows principal `cluster.local/ns/default/sa/sleep` to access the workload
+using `GET` method when the request includes a `version` header of value `v1` or `v2`.
+Any requests not matched with the policy will be denied by default.
+
+That's all you need to enable access control for a workload, compared to the `v1alpha1`
+RBAC policy, you would need to configure three resources to achieve the same result:
```

Comment:
Is it true the example below (v1alpha1) doesn't specify the rbac policy is applied to httpbin v1 service?   it is instead applied to httpbin services regardless of versions.  If so, this is worth highlighting.

Reply:
This depends on the service definition, here we are assuming the `httpbin` service is defined like the following:
```
apiVersion: v1
kind: Service
metadata:
  name: httpbin
  namespace: foo
spec:
  selector:
    app: httpbin
    version: v1
  ports:
    # omitted
```

Updated the doc to make this part more clear.